### ISSUE TITLE
How import module from Node?

### ISSUE THREAD

**Issue by gretmn102** [fable-compiler/fable2-samples#50]
Hello, friends. Help me please, because I don't know what to do. I tried:
```fsharp
type [<Import("Disposable","vscode")>] Disposable(callOnDispose: JsFunc) =
    static member from([<ParamArray>] disposableLikes: obj[]): Disposable = failwith "JS only"
    member __.dispose(): obj = failwith "JS only"
```

But `npm run build` given me error:
```
ERROR in ./src/Fable.Import.VSCode.fs
Module not found: Error: Can't resolve 'vscode' in 'E:\Project\fable2-samples\nodejsbundle\src'
 @ ./src/Fable.Import.VSCode.fs 8:14-31
 @ ./src/App.fs
 @ ./src/App.fsproj
```

[Here](https://github.com/gretmn102/fable2-samples/commit/97de13f5ea1e1de44e1b3b079b7c65056fdf471b) is my code cloned from your sample repository. What am I doing wrong?

**Comment by alfonsogarciacaro** [comment]
Did you run "npm install"?

**Comment by gretmn102** [comment]
Yes, of course
[output.txt](https://github.com/fable-compiler/fable2-samples/files/5073044/output.txt)

**Comment by alfonsogarciacaro** [comment]
Hmm, I did try your code and got the same error, but I'm not sure what's the reason. I guess the vscode module needs to be resolved differently. To be honest, I don't know what Fable.Import.VSCode.fs is doing in that example. You should be able to remove it. Are you interested in writing a VS Code plugin with Fable? I think at one point we had an example/tutorial of doing that but it's been a while and I don't know f they've changed the API.

**Comment by gretmn102** [comment]
> Are you interested in writing a VS Code plugin with Fable?
Yes. [It's](https://github.com/fable-compiler/fable-import/blob/master/VSCode/Fable.Import.VSCode.fs) very very old, and I wonted upgrade it. It uses [this](https://github.com/fable-compiler/fable-import/blob/master/VSCode/Fable.Import.VSCode.fs#L45) principle of import, and I naively wanted to repeat it. But it is also logical that somehow I can import modules from Node into Fable. It works in old versions, but how to do it in the new one? Thanks for your response. Fable awesome thing what I ever seen :+1:
